exercises/fizz-buzz/README.md

```diff
@@ -0,0 +1,29 @@
+# Fizz Buzz
+
+Print numbers from 1 to n with the following exceptions:
+
+1. When a number is a multiple of 3, then print "Fizz"
+2. When a number is a multiple of 5, then print "Buzz"
+3. When a number is both a multiple of 3 and a multiple of 5, then print "Fizz Buzz"
+
+The output should be a vector of strings, such as, "1" "2" "Fizz" "4" "Buzz" ...
```

Comment:
Would it be too much of a give-away to put `c("1","2","Fizz","4","Buzz", ...)` here? As a hint for the technically & idiomatically correct answeR?

Reply:
I think this is somewhat necessary. Don't know how a beginner may go about this. But certainly, they'd have to know that the numbers are strings. Otherwise, they'd go down a rabbit-hole of checking that numbers are numbers and (Fizz, Buzz) are strings. Which is impossible in R at least when returned as a vector

Reply:
Although I did already mention that the output sould be a vector of strings... hhmmm. I am fine keeping it just as is. Unless you have a strong opinion or there is some clear instruction.

Reply:
I have no strong opinion. And they can read the tests, so let's keep it like this.